libs/portals/form-system/graphql/src/lib/client.ts

```diff
@@ -0,0 +1,35 @@
+import { RetryLink } from '@apollo/client/link/retry'
+import { onError } from '@apollo/client/link/error'
+import fetch from 'cross-fetch'
+import {
+  ApolloClient,
+  InMemoryCache,
+  HttpLink,
+  ApolloLink,
+} from '@apollo/client'
+
+const retryLink = new RetryLink()
+
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors)
+    graphQLErrors.map(({ message, locations, path, extensions }) => {
+      const problem = JSON.stringify(extensions?.problem, null, '  ')
+      console.log(
+        `[GraphQL error]: Message: ${message}, Path: ${path}, Problem: ${problem}`,
+      )
+    })
+
+  if (networkError) console.log(`[Network error]: ${networkError}`)
+})
+
+export const initializeClient = (baseApiUrl: string) => {
```

Comment:
Let's move this to the webapp, so we don't accidentally end up with two apollo client instances in admin-portal.